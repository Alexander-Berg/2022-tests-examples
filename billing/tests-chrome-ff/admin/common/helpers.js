const oebsStatusSelector = '.src-common-components-Person-___person-module__person-row';
const typeIdSelector = '.src-common-components-Person-___person-module__person-id__type-id';
const personName = '.yb-person-id__name';

module.exports.hideElements = [oebsStatusSelector, typeIdSelector];
module.exports.ignoreElements = [personName];
module.exports.personModalSelector = '.yb-person-modal';
