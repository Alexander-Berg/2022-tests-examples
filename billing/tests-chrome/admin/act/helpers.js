const assert = require('chai').assert;

module.exports.saveGoodDebt = async function (browser) {
    await browser.ybSetSteps('Признает долг хорошим');
    await browser.click('#label-act-saver__checkbox-is-good-debt');
    await browser.click('#act-saver__btn-save-good-debt');
    await browser.waitForVisible('span=Изменение учета долга сохранено');
};

module.exports.hideElements = [
    '.yb-copyable-header__external-id',
    '.src-common-components-Table-___table-module__table-container',
    '.src-common-components-ExportState-___styles-module__reexportStatus'
];

module.exports.reexportStatusSelector =
    '.src-common-components-ExportState-___styles-module__reexportStatus';

module.exports.assertPageEmpty = async function (browser) {
    await browser.ybSetSteps('Проверяет, что страница пуста');
    const text = browser.getText('.yb-content');
    assert(text, 'Эксперимент UI', 'Страница не пуста');
};
