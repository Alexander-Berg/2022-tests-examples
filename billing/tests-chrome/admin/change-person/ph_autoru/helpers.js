module.exports.changePersonTypeToPhAutoru = async function (browser) {
    const url = await browser.url();
    await browser.ybAbsoluteUrl(url.replace('type=ph', `type=ph_autoru`));
};
