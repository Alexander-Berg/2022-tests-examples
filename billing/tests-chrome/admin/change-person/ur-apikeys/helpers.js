const { partner } = require('./common');

module.exports.changePersonTypeToUrApikeys = async function (browser) {
    const url = await browser.url();
    await browser.ybAbsoluteUrl(url.replace('type=ur', `type=ur&from_service=apikeys`));
};

module.exports.navigateToNewUrApikeysChangePerson = async function (browser, clientId) {
    await browser.ybUrl(
        'admin',
        `change-person.xml?type=ur&from_service=apikeys&partner=${partner}&client_id=${clientId}`
    );
};
