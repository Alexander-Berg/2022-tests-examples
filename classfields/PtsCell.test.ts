import { createHelpers, makeFullReport } from 'app/server/tmpl/android/v1/Main';
import { AppConfig } from 'app/server/tmpl/android/v1/helpers/AppConfig';
import type { Request } from 'express';
import type { PtsBlock, RawVinReport } from '@vertis/schema-registry/ts-types-snake/auto/api/vin/vin_report_model';
import { PtsBlock_PtsType } from '@vertis/schema-registry/ts-types-snake/auto/api/vin/vin_report_model';
import { response200ForVin } from 'app/server/resources/publicApi/methods/getCarfaxOfferReportRaw.fixtures';
import { yogaToXml } from 'app/server/tmpl/XMLBuilder';
import { Status } from '@vertis/schema-registry/ts-types-snake/auto/api/vin/vin_resolution_enums';
import type { Photo } from '@vertis/schema-registry/ts-types-snake/auto/api/vin/common';
import type { VinReportComment } from '@vertis/schema-registry/ts-types-snake/auto/api/vin/comments/comment_api_model';

let req: Request;
beforeEach(() => {
    req = {
        headers: {
            'x-android-app-version': '10.0.0',
        },
    } as unknown as Request;
    jest.spyOn(Math, 'random').mockReturnValue(0);
});

describe('pts cell', () => {
    it('block not exists', () => {
        const report = response200ForVin('Z8T4DNFUCDM014995').report as RawVinReport;
        report.pts_info = undefined;

        const result = render(report, req);
        const pts = result.find((node) => node.id === 'pts');

        expect(pts).toBeUndefined();
    });

    it('block exists', () => {
        const report = response200ForVin('Z8T4DNFUCDM014995').report as RawVinReport;
        report.pts_info = buildPtsItem(false);
        report.pts_info.pts_type = PtsBlock_PtsType.DIGITAL;
        report.comments = [
            {
                id: '65a70a4521cd41a2b4cc126f627c3116',
                block_id: 'pts',
                text: 'Это комментарий с тремя фотографиями и текстом, который не помещается в одну строку.',
                photos: [
                    {
                        mds_photo_info: {
                            namespace: 'autoru-reviews',
                            group_id: 69673,
                            name: 'd228e9ddda7df5c6de38ad38e1ed4102',
                        },
                        sizes: {
                            thumb_m: '//images.mds-proxy.test.avto.ru/get-autoru-reviews/69673/d228e9ddda7df5c6de38ad38e1ed4102/thumb_m',
                        },
                        is_deleted: false,
                    } as Photo,
                ],
                user: {
                    name: 'Продавец',
                    current_user_is_owner: true,
                },
                create_time: '1594240743610',
                update_time: '1594240751710',
                actions: {
                    edit: true,
                    'delete': true,
                },
                is_deleted: false,
            } as VinReportComment,
        ];

        const result = render(report, req);
        const pts = result.find((node) => node.id === 'pts');
        const xml = pts ? yogaToXml([ pts ]) : '';

        expect(pts).toMatchSnapshot();
        expect(xml).toMatchSnapshot();
    });

});

function render(report: RawVinReport, req: Request) {
    const appConfig = new AppConfig({}, req);
    const helpers = createHelpers(report, appConfig);
    return makeFullReport(report, helpers);
}

function buildPtsItem(
    isUpdating: boolean,
): PtsBlock {
    return {
        header: {
            title: 'Данные из ПТС',
            is_updating: isUpdating,
            timestamp_update: '1595405273648',
        },
        vin: 'WBADE210X0BM83317',
        mark: {
            key: 'Марка',
            value: 'BMW',
            value_text: 'BMW',
            status: Status.OK,
        },
        color: {
            key: 'Цвет',
            value: 'Зеленый',
            value_text: 'Зеленый',
            status: Status.OK,
        },
        year: {
            key: 'Год выпуска',
            value: 1997,
            value_text: '1997 г.',
            status: Status.OK,
        },
        horse_power: {
            key: 'Мощность двигателя',
            value: 234,
            value_text: '234 л.с.',
            status: Status.OK,
        },
        displacement: {
            key: 'Объём двигателя',
            value: 3500,
            value_text: '3,5 л',
            status: Status.OK,
        },
        mark_logo: {
            name: 'standalone_preview_mark_icon',
            sizes: {
                orig: '//avatars.mds.yandex.net/get-verba/997355/2a00000170a99124c3a521a92b8cd4846660/dealer_logo',
            },
        },
        commentable: {
            block_id: 'pts',
            add_comment: false,
        },
        status: Status.OK,
    } as unknown as PtsBlock;
}
