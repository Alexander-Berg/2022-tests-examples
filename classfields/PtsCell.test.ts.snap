// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pts cell block exists 1`] = `
Object {
  "backgroundColor": "FFFFFF",
  "backgroundColorDark": "1C1C1E",
  "backgroundColorID": 45,
  "children": Array [
    Object {
      "children": Array [
        Object {
          "children": Array [
            Object {
              "flex": 1,
              "fontSize": 24,
              "fontWeight": "bold",
              "name": "text",
              "text": "Данные из ПТС",
              "textColor": "000000",
              "textColorDark": "FFFFFF",
              "textColorID": 47,
            },
            Object {
              "alignItems": "center",
              "children": Array [
                Object {
                  "name": "ignore",
                },
                Object {
                  "fontSize": 14,
                  "name": "text",
                  "onTap": undefined,
                  "text": "Обновлено 22 июля 2020",
                  "textColor": "80000000",
                  "textColorDark": "98FFFFFF",
                  "textColorID": 48,
                },
              ],
              "margin": "{3, 0, 0, 0}",
              "name": "stack",
            },
          ],
          "flex": 1,
          "flexDirection": "column",
          "name": "stack",
          "padding": "{0, 24, 0, 24}",
        },
        Object {
          "name": "ignore",
        },
      ],
      "name": "stack",
    },
    Object {
      "children": Array [
        Object {
          "children": Array [
            Object {
              "flex": "3",
              "fontSize": "14",
              "margin": "{0, 0, 0, 16}",
              "name": "text",
              "text": "Электронный ПТС",
              "textColor": "80000000",
              "textColorDark": "98FFFFFF",
              "textColorID": 48,
            },
            Object {
              "flex": "5",
              "fontSize": "14",
              "name": "text",
              "text": "Есть",
              "textColor": "2276D3",
              "textColorDark": "4AA6FF",
              "textColorID": 49,
            },
            Object {
              "children": Array [
                Object {
                  "margin": "{0,16,0,16}",
                  "name": "text",
                  "text": "Вам необходимо самостоятельно переоформить машину в системе электронных паспортов после подписания договора. Это просто.",
                  "textColor": "000000",
                  "textColorDark": "FFFFFF",
                  "textColorID": 47,
                  "textSize": 16,
                },
                Object {
                  "margin": "{16,16,24,16}",
                  "name": "text",
                  "onTap": "https://mag.auto.ru/article/eptsexplain/",
                  "text": "Подробнее об электронных ПТС",
                  "textColor": "2276D3",
                  "textColorDark": "4AA6FF",
                  "textColorID": 49,
                  "textSize": 16,
                },
              ],
              "flexDirection": "column",
              "name": "onTapStack",
              "title": "Электронный ПТС",
            },
          ],
          "id": "pts_type",
          "margin": "{8, 0, 8, 0}",
          "name": "stack",
        },
        Object {
          "children": Array [
            Object {
              "flex": "3",
              "fontSize": "14",
              "margin": "{0, 0, 0, 16}",
              "name": "text",
              "text": "VIN",
              "textColor": "80000000",
              "textColorDark": "98FFFFFF",
              "textColorID": 48,
            },
            Object {
              "flex": "5",
              "fontSize": "14",
              "name": "text",
              "text": "WBADE210X0BM83317",
              "textColor": "000000",
              "textColorDark": "FFFFFF",
              "textColorID": 47,
              "textSelectable": true,
            },
          ],
          "margin": "{8, 0, 8, 0}",
          "name": "stack",
        },
        Object {
          "children": Array [
            Object {
              "flex": "3",
              "fontSize": "14",
              "margin": "{0, 0, 0, 16}",
              "name": "text",
              "text": "Год выпуска",
              "textColor": "80000000",
              "textColorDark": "98FFFFFF",
              "textColorID": 48,
            },
            Object {
              "flex": "5",
              "fontSize": "14",
              "name": "text",
              "text": "1997",
              "textColor": "000000",
              "textColorDark": "FFFFFF",
              "textColorID": 47,
              "textSelectable": undefined,
            },
          ],
          "margin": "{8, 0, 8, 0}",
          "name": "stack",
        },
        Object {
          "children": Array [
            Object {
              "flex": "3",
              "fontSize": "14",
              "margin": "{0, 0, 0, 16}",
              "name": "text",
              "text": "Объём двигателя",
              "textColor": "80000000",
              "textColorDark": "98FFFFFF",
              "textColorID": 48,
            },
            Object {
              "flex": "5",
              "fontSize": "14",
              "name": "text",
              "text": "3,5 л",
              "textColor": "000000",
              "textColorDark": "FFFFFF",
              "textColorID": 47,
              "textSelectable": undefined,
            },
          ],
          "margin": "{8, 0, 8, 0}",
          "name": "stack",
        },
        Object {
          "children": Array [
            Object {
              "flex": "3",
              "fontSize": "14",
              "margin": "{0, 0, 0, 16}",
              "name": "text",
              "text": "Мощность двигателя",
              "textColor": "80000000",
              "textColorDark": "98FFFFFF",
              "textColorID": 48,
            },
            Object {
              "flex": "5",
              "fontSize": "14",
              "name": "text",
              "text": "234 л.с.",
              "textColor": "000000",
              "textColorDark": "FFFFFF",
              "textColorID": 47,
              "textSelectable": undefined,
            },
          ],
          "margin": "{8, 0, 8, 0}",
          "name": "stack",
        },
        Object {
          "children": Array [
            Object {
              "flex": "3",
              "fontSize": "14",
              "margin": "{0, 0, 0, 16}",
              "name": "text",
              "text": "Цвет",
              "textColor": "80000000",
              "textColorDark": "98FFFFFF",
              "textColorID": 48,
            },
            Object {
              "flex": "5",
              "fontSize": "14",
              "name": "text",
              "text": "Зеленый",
              "textColor": "000000",
              "textColorDark": "FFFFFF",
              "textColorID": 47,
              "textSelectable": undefined,
            },
          ],
          "margin": "{8, 0, 8, 0}",
          "name": "stack",
        },
      ],
      "flexDirection": "column",
      "name": "stack",
      "padding": "{8, 24, 0, 24}",
    },
    Object {
      "children": Array [
        Object {
          "backgroundColor": "1A000000",
          "backgroundColorDark": "1AFFFFFF",
          "backgroundColorID": 45,
          "height": 1,
          "margin": "{8,0,8,0}",
          "name": "stack",
        },
        Object {
          "alignItems": "center",
          "children": Array [
            Object {
              "aspectRatio": "1",
              "height": "24",
              "margin": "{0, 0, 0, 24}",
              "name": "image",
              "url": "https://yastatic.net/s3/vertis-frontend/autoru-mobile/carfax-icons/android/checked/light.png",
              "urlDark": "https://yastatic.net/s3/vertis-frontend/autoru-mobile/carfax-icons/android/checked/dark.png",
            },
            Object {
              "flex": "1",
              "fontSize": "14",
              "name": "text",
              "text": "Данные из ПТС совпадают с указанными в объявлении",
              "textColor": "000000",
              "textColorDark": "FFFFFF",
              "textColorID": 47,
            },
          ],
          "minHeight": 56,
          "name": "stack",
          "padding": "{0,0,0,0}",
        },
        Object {
          "name": "ignore",
        },
      ],
      "flexDirection": "column",
      "margin": "{0, 24, 0, 24}",
      "name": "stack",
    },
    Object {
      "children": Array [
        Object {
          "children": Array [
            Object {
              "children": Array [
                Object {
                  "children": Array [
                    Object {
                      "children": Array [
                        Object {
                          "children": Array [
                            Object {
                              "alignSelf": "flexEnd",
                              "blurPreview": false,
                              "height": "24",
                              "name": "image",
                              "positionType": "absolute",
                              "previewBytes": "iVBORw0KGgoAAAANSUhEUgAAAFAAAABgCAYAAACKa/UAAAAACXBIWXMAACxLAAAsSwGlPZapAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAR/SURBVHgB7Zz/VRNBEMdnN2CICZh0ECoQKxAqUCpQO8AKjBWoFYgVqBWYDsQKTAeAHCThkh13lxzckYTc7u3eXbj5vMfjr+S9fPOdnR83GxZcjRAeMa2nWww8wuXfAAhrOBCZIAdmhCOwMyCs4YjiHAhrOKMQzoR0IAmYBc5IwEzwyQYJmAXeaTQGgECZ2JKokB4AYYUWUAD+AcIKLaBMxSdAWHHjQIEDIKzQAk43OTnQEi0gZWJ7bsdZyFgfCGPu5oEoKBNbcCeggD4QxtwKOJmMKJFYcCtgp9NRSaQPhBGJZyIo6Bw0JflQCdkPIIxICKjPQaoHjUgIqM9BBpRMDJh/LizETyBSMydgWKNz0IQ5AXVfTAPW1Cxc7UDEb0CkYvFuDLV1qVm6+hVcjf7Kf11Yc/JYb1sIhXE6lq+3URinYqmA29uNPtBwYSUPL1hSUb2SBwUMw/ExEA/yoIA0I1zNyh1pnOJHIJaSqkZa55qwsDowDtWEy0kl4OR69JkGrYtJJaBKJgj4BYg5Ul+0mXA4BmKO1ALO5oR9IBIYXfWikmYeIwGpP57H+LIhuTCJsYDkwiRW113JhXdYCUguvMP6wnXI8B0Q9gKquhCFqHx3kunK/yQc96reI2cSUPXIUwaVTihOZmXB5ei3fKc9KCGlmAeuAgW+h4riREBV1lQ1oTgRUFHVhOJMQJVQBJYvlIfDYRc84kxAxU6rcQwV61CcCqjQHUqJQllw/hw84lxA1aGI6bQ0bZ4cfOyCR5wLqNjZaf4oS1ZG4fd5thcBFTorl2DXmnFYrxCO0Hs1YnJYgvPQa4fkTUBFq9U6KUGv3PZZyngVUPHs6dbnos/DKbLX4AnvAipmXUphV8gYY97OQa+TijinMow2kf2CYra8zuRUpgMeyMWBCr3ZUFxSaV9cDPfBA7kJqFBJpbB+mfk5B3ML4TjnV6OjGsAnyBcvYZyrAyN0Zsbcny17CeNCBFRsNxu9vEVkNfYBHFNICMe5uBz2ZJnh/IMtIxzXZZPk7qefC3NgRN5O3HgyOgKHFO7AiBydeCZduOvKhYU7MCJHJ7Y362NnX1RpHBiRlxPDMUoX6rXlTJTGgRHKiWIy9d6xbNbZV3BA6RwYkUfvLJAd7jTrmX6lpLQCKmYiKqfsgx8yJ5TShXAcNYCQ7deBx+SiEsp3yECpBYzQGXqKB+DnGct+cGVfG5Y6hO+jQromoMcZewOOUV/QbHXZiLUSMOJfMHzLuS51uuAOeR7iC9PSZi1C+D5qhSRkeCDcXsOV5yH7dXpq9gBqLR0YR2dqwb47XPAcSCcepHXiWjowjs7Uza0XQuhbAwPITtfEiWvvwPs4PB8HMrG8W5VYHp2AEWr6PBug7kMGZF/eazbqS+vQRytghDojN5AdyQ/6Cmxdiew4vBYfF52Lj17AOMqVyOGtrCNfgrmYA2C812o8SWT+SgkYJwiCPQG1vRpnr/BmAamb6oUIJ+E1HkZurKyA9zlFbG8Eoz2o6TWQXSlMtA7SRYQ2Y9BOvECGNQrx7T8eFLK+RVy2nwAAAABJRU5ErkJggg==",
                              "url": "https://yastatic.net/s3/vertis-frontend/autoru-mobile/carfax-icons/android/comment-tail/light.png",
                              "urlDark": "https://yastatic.net/s3/vertis-frontend/autoru-mobile/carfax-icons/android/comment-tail/dark.png",
                              "width": "20",
                            },
                            Object {
                              "backgroundColor": "F4F6F8",
                              "backgroundColorDark": "2C2C2E",
                              "backgroundColorID": 46,
                              "children": Array [
                                Object {
                                  "alignItems": "center",
                                  "children": Array [
                                    Object {
                                      "fontSize": "12",
                                      "fontWeight": "medium",
                                      "margin": "{0, 0, 0, 16}",
                                      "name": "text",
                                      "text": "Ваш комментарий",
                                      "textColor": "80000000",
                                      "textColorDark": "98FFFFFF",
                                      "textColorID": 48,
                                    },
                                    Object {
                                      "fontSize": "12",
                                      "name": "text",
                                      "text": "8 июля 2020",
                                      "textColor": "80000000",
                                      "textColorDark": "98FFFFFF",
                                      "textColorID": 48,
                                    },
                                  ],
                                  "justifyContent": "spaceBetween",
                                  "name": "stack",
                                  "padding": "{0, 16, 0, 16}",
                                },
                                Object {
                                  "children": Array [
                                    Object {
                                      "fontSize": "14",
                                      "name": "text",
                                      "text": "Это комментарий с тремя фотографиями и текстом, который не помещается в одну строку.",
                                      "textColor": "000000",
                                      "textColorDark": "FFFFFF",
                                      "textColorID": 47,
                                    },
                                  ],
                                  "flex": 1,
                                  "name": "stack",
                                  "padding": "{10, 16, 12, 16}",
                                },
                                Object {
                                  "aspectRatio": "1.3",
                                  "contentMode": "scaleAspectFill",
                                  "flex": "1",
                                  "id": "autoru-reviews:69673-d228e9ddda7df5c6de38ad38e1ed4102",
                                  "name": "image",
                                  "onTapAction": "showFull",
                                },
                              ],
                              "cornerRadius": "16",
                              "flex": "1",
                              "flexDirection": "column",
                              "name": "stack",
                              "padding": "{14, 0, 0, 0}",
                            },
                          ],
                          "flex": "1",
                          "flexDirection": "row",
                          "id": "pts",
                          "justifyContent": "flexEnd",
                          "margin": "{8, 16, 8, 16}",
                          "name": "stack",
                          "onTapAction": "editComment",
                          "padding": "{0, 6, 0, 6}",
                        },
                        Object {
                          "children": Array [
                            Object {
                              "fontSize": "14",
                              "name": "text",
                              "text": "Изменить",
                              "textColor": "2276D3",
                              "textColorDark": "4AA6FF",
                              "textColorID": 49,
                            },
                          ],
                          "flexDirection": "row",
                          "id": "pts",
                          "justifyContent": "flexEnd",
                          "margin": "{0, 24, 8, 24}",
                          "name": "stack",
                          "onTapAction": "editComment",
                        },
                      ],
                      "flexDirection": "column",
                      "name": "stack",
                    },
                  ],
                  "flexDirection": "column",
                  "name": "stack",
                },
              ],
              "flexDirection": "column",
              "name": "stack",
            },
          ],
          "flexDirection": "column",
          "margin": "{0, 0, 0, 0}",
          "name": "stack",
        },
      ],
      "flexDirection": "column",
      "name": "stack",
    },
  ],
  "cornerRadius": "24",
  "flexDirection": "column",
  "id": "pts",
  "margin": "{0, 0, 16, 0}",
  "name": "stack",
  "padding": "{24, 0, 24, 0}",
}
`;

exports[`pts cell block exists 2`] = `"<list><stack backgroundColor=\\"FFFFFF\\" backgroundColorDark=\\"1C1C1E\\" backgroundColorID=\\"45\\" cornerRadius=\\"24\\" flexDirection=\\"column\\" id=\\"pts\\" margin=\\"{0, 0, 16, 0}\\" padding=\\"{24, 0, 24, 0}\\"><stack><stack flexDirection=\\"column\\" padding=\\"{0, 24, 0, 24}\\" flex=\\"1\\"><text flex=\\"1\\" fontSize=\\"24\\" fontWeight=\\"bold\\" text=\\"Данные из ПТС\\" textColor=\\"000000\\" textColorDark=\\"FFFFFF\\" textColorID=\\"47\\"/><stack margin=\\"{3, 0, 0, 0}\\" alignItems=\\"center\\"><ignore/><text fontSize=\\"14\\" text=\\"Обновлено 22 июля 2020\\" textColor=\\"80000000\\" textColorDark=\\"98FFFFFF\\" textColorID=\\"48\\"/></stack></stack><ignore/></stack><stack flexDirection=\\"column\\" padding=\\"{8, 24, 0, 24}\\"><stack id=\\"pts_type\\" margin=\\"{8, 0, 8, 0}\\"><text flex=\\"3\\" fontSize=\\"14\\" margin=\\"{0, 0, 0, 16}\\" text=\\"Электронный ПТС\\" textColor=\\"80000000\\" textColorDark=\\"98FFFFFF\\" textColorID=\\"48\\"/><text flex=\\"5\\" fontSize=\\"14\\" text=\\"Есть\\" textColor=\\"2276D3\\" textColorDark=\\"4AA6FF\\" textColorID=\\"49\\"/><onTapStack flexDirection=\\"column\\" title=\\"Электронный ПТС\\"><text text=\\"Вам необходимо самостоятельно переоформить машину в&#160;системе электронных паспортов после подписания договора. Это просто.\\" textSize=\\"16\\" margin=\\"{0,16,0,16}\\" textColor=\\"000000\\" textColorDark=\\"FFFFFF\\" textColorID=\\"47\\"/><text text=\\"Подробнее об электронных ПТС\\" textSize=\\"16\\" margin=\\"{16,16,24,16}\\" onTap=\\"https://mag.auto.ru/article/eptsexplain/\\" textColor=\\"2276D3\\" textColorDark=\\"4AA6FF\\" textColorID=\\"49\\"/></onTapStack></stack><stack margin=\\"{8, 0, 8, 0}\\"><text flex=\\"3\\" fontSize=\\"14\\" margin=\\"{0, 0, 0, 16}\\" text=\\"VIN\\" textColor=\\"80000000\\" textColorDark=\\"98FFFFFF\\" textColorID=\\"48\\"/><text flex=\\"5\\" fontSize=\\"14\\" text=\\"WBADE210X0BM83317\\" textSelectable=\\"true\\" textColor=\\"000000\\" textColorDark=\\"FFFFFF\\" textColorID=\\"47\\"/></stack><stack margin=\\"{8, 0, 8, 0}\\"><text flex=\\"3\\" fontSize=\\"14\\" margin=\\"{0, 0, 0, 16}\\" text=\\"Год выпуска\\" textColor=\\"80000000\\" textColorDark=\\"98FFFFFF\\" textColorID=\\"48\\"/><text flex=\\"5\\" fontSize=\\"14\\" text=\\"1997\\" textColor=\\"000000\\" textColorDark=\\"FFFFFF\\" textColorID=\\"47\\"/></stack><stack margin=\\"{8, 0, 8, 0}\\"><text flex=\\"3\\" fontSize=\\"14\\" margin=\\"{0, 0, 0, 16}\\" text=\\"Объём двигателя\\" textColor=\\"80000000\\" textColorDark=\\"98FFFFFF\\" textColorID=\\"48\\"/><text flex=\\"5\\" fontSize=\\"14\\" text=\\"3,5&#160;л\\" textColor=\\"000000\\" textColorDark=\\"FFFFFF\\" textColorID=\\"47\\"/></stack><stack margin=\\"{8, 0, 8, 0}\\"><text flex=\\"3\\" fontSize=\\"14\\" margin=\\"{0, 0, 0, 16}\\" text=\\"Мощность двигателя\\" textColor=\\"80000000\\" textColorDark=\\"98FFFFFF\\" textColorID=\\"48\\"/><text flex=\\"5\\" fontSize=\\"14\\" text=\\"234&#160;л.с.\\" textColor=\\"000000\\" textColorDark=\\"FFFFFF\\" textColorID=\\"47\\"/></stack><stack margin=\\"{8, 0, 8, 0}\\"><text flex=\\"3\\" fontSize=\\"14\\" margin=\\"{0, 0, 0, 16}\\" text=\\"Цвет\\" textColor=\\"80000000\\" textColorDark=\\"98FFFFFF\\" textColorID=\\"48\\"/><text flex=\\"5\\" fontSize=\\"14\\" text=\\"Зеленый\\" textColor=\\"000000\\" textColorDark=\\"FFFFFF\\" textColorID=\\"47\\"/></stack></stack><stack flexDirection=\\"column\\" margin=\\"{0, 24, 0, 24}\\"><stack backgroundColor=\\"1A000000\\" backgroundColorDark=\\"1AFFFFFF\\" backgroundColorID=\\"45\\" height=\\"1\\" margin=\\"{8,0,8,0}\\"/><stack alignItems=\\"center\\" minHeight=\\"56\\" padding=\\"{0,0,0,0}\\"><image aspectRatio=\\"1\\" height=\\"24\\" margin=\\"{0, 0, 0, 24}\\" url=\\"https://yastatic.net/s3/vertis-frontend/autoru-mobile/carfax-icons/android/checked/light.png\\" urlDark=\\"https://yastatic.net/s3/vertis-frontend/autoru-mobile/carfax-icons/android/checked/dark.png\\"/><text flex=\\"1\\" fontSize=\\"14\\" text=\\"Данные из&#160;ПТС совпадают с&#160;указанными в&#160;объявлении\\" textColor=\\"000000\\" textColorDark=\\"FFFFFF\\" textColorID=\\"47\\"/></stack><ignore/></stack><stack flexDirection=\\"column\\"><stack flexDirection=\\"column\\" margin=\\"{0, 0, 0, 0}\\"><stack flexDirection=\\"column\\"><stack flexDirection=\\"column\\"><stack flexDirection=\\"column\\"><stack flex=\\"1\\" flexDirection=\\"row\\" id=\\"pts\\" justifyContent=\\"flexEnd\\" margin=\\"{8, 16, 8, 16}\\" onTapAction=\\"editComment\\" padding=\\"{0, 6, 0, 6}\\"><image width=\\"20\\" alignSelf=\\"flexEnd\\" blurPreview=\\"false\\" height=\\"24\\" positionType=\\"absolute\\" previewBytes=\\"iVBORw0KGgoAAAANSUhEUgAAAFAAAABgCAYAAACKa/UAAAAACXBIWXMAACxLAAAsSwGlPZapAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAR/SURBVHgB7Zz/VRNBEMdnN2CICZh0ECoQKxAqUCpQO8AKjBWoFYgVqBWYDsQKTAeAHCThkh13lxzckYTc7u3eXbj5vMfjr+S9fPOdnR83GxZcjRAeMa2nWww8wuXfAAhrOBCZIAdmhCOwMyCs4YjiHAhrOKMQzoR0IAmYBc5IwEzwyQYJmAXeaTQGgECZ2JKokB4AYYUWUAD+AcIKLaBMxSdAWHHjQIEDIKzQAk43OTnQEi0gZWJ7bsdZyFgfCGPu5oEoKBNbcCeggD4QxtwKOJmMKJFYcCtgp9NRSaQPhBGJZyIo6Bw0JflQCdkPIIxICKjPQaoHjUgIqM9BBpRMDJh/LizETyBSMydgWKNz0IQ5AXVfTAPW1Cxc7UDEb0CkYvFuDLV1qVm6+hVcjf7Kf11Yc/JYb1sIhXE6lq+3URinYqmA29uNPtBwYSUPL1hSUb2SBwUMw/ExEA/yoIA0I1zNyh1pnOJHIJaSqkZa55qwsDowDtWEy0kl4OR69JkGrYtJJaBKJgj4BYg5Ul+0mXA4BmKO1ALO5oR9IBIYXfWikmYeIwGpP57H+LIhuTCJsYDkwiRW113JhXdYCUguvMP6wnXI8B0Q9gKquhCFqHx3kunK/yQc96reI2cSUPXIUwaVTihOZmXB5ei3fKc9KCGlmAeuAgW+h4riREBV1lQ1oTgRUFHVhOJMQJVQBJYvlIfDYRc84kxAxU6rcQwV61CcCqjQHUqJQllw/hw84lxA1aGI6bQ0bZ4cfOyCR5wLqNjZaf4oS1ZG4fd5thcBFTorl2DXmnFYrxCO0Hs1YnJYgvPQa4fkTUBFq9U6KUGv3PZZyngVUPHs6dbnos/DKbLX4AnvAipmXUphV8gYY97OQa+TijinMow2kf2CYra8zuRUpgMeyMWBCr3ZUFxSaV9cDPfBA7kJqFBJpbB+mfk5B3ML4TjnV6OjGsAnyBcvYZyrAyN0Zsbcny17CeNCBFRsNxu9vEVkNfYBHFNICMe5uBz2ZJnh/IMtIxzXZZPk7qefC3NgRN5O3HgyOgKHFO7AiBydeCZduOvKhYU7MCJHJ7Y362NnX1RpHBiRlxPDMUoX6rXlTJTGgRHKiWIy9d6xbNbZV3BA6RwYkUfvLJAd7jTrmX6lpLQCKmYiKqfsgx8yJ5TShXAcNYCQ7deBx+SiEsp3yECpBYzQGXqKB+DnGct+cGVfG5Y6hO+jQromoMcZewOOUV/QbHXZiLUSMOJfMHzLuS51uuAOeR7iC9PSZi1C+D5qhSRkeCDcXsOV5yH7dXpq9gBqLR0YR2dqwb47XPAcSCcepHXiWjowjs7Uza0XQuhbAwPITtfEiWvvwPs4PB8HMrG8W5VYHp2AEWr6PBug7kMGZF/eazbqS+vQRytghDojN5AdyQ/6Cmxdiew4vBYfF52Lj17AOMqVyOGtrCNfgrmYA2C812o8SWT+SgkYJwiCPQG1vRpnr/BmAamb6oUIJ+E1HkZurKyA9zlFbG8Eoz2o6TWQXSlMtA7SRYQ2Y9BOvECGNQrx7T8eFLK+RVy2nwAAAABJRU5ErkJggg==\\" url=\\"https://yastatic.net/s3/vertis-frontend/autoru-mobile/carfax-icons/android/comment-tail/light.png\\" urlDark=\\"https://yastatic.net/s3/vertis-frontend/autoru-mobile/carfax-icons/android/comment-tail/dark.png\\"/><stack backgroundColor=\\"F4F6F8\\" backgroundColorDark=\\"2C2C2E\\" backgroundColorID=\\"46\\" cornerRadius=\\"16\\" flex=\\"1\\" flexDirection=\\"column\\" padding=\\"{14, 0, 0, 0}\\"><stack alignItems=\\"center\\" justifyContent=\\"spaceBetween\\" padding=\\"{0, 16, 0, 16}\\"><text fontSize=\\"12\\" fontWeight=\\"medium\\" margin=\\"{0, 0, 0, 16}\\" text=\\"Ваш комментарий\\" textColor=\\"80000000\\" textColorDark=\\"98FFFFFF\\" textColorID=\\"48\\"/><text fontSize=\\"12\\" text=\\"8 июля 2020\\" textColor=\\"80000000\\" textColorDark=\\"98FFFFFF\\" textColorID=\\"48\\"/></stack><stack flex=\\"1\\" padding=\\"{10, 16, 12, 16}\\"><text fontSize=\\"14\\" text=\\"Это комментарий с тремя фотографиями и текстом, который не помещается в одну строку.\\" textColor=\\"000000\\" textColorDark=\\"FFFFFF\\" textColorID=\\"47\\"/></stack><image aspectRatio=\\"1.3\\" contentMode=\\"scaleAspectFill\\" flex=\\"1\\" id=\\"autoru-reviews:69673-d228e9ddda7df5c6de38ad38e1ed4102\\" onTapAction=\\"showFull\\"/></stack></stack><stack flexDirection=\\"row\\" id=\\"pts\\" justifyContent=\\"flexEnd\\" margin=\\"{0, 24, 8, 24}\\" onTapAction=\\"editComment\\"><text fontSize=\\"14\\" text=\\"Изменить\\" textColor=\\"2276D3\\" textColorDark=\\"4AA6FF\\" textColorID=\\"49\\"/></stack></stack></stack></stack></stack></stack></stack></list>"`;
